pegasus/helpers/section_api_helpers.rb

```diff
@@ -213,6 +213,29 @@ def self.valid_grade?(grade)
     valid_grades.include? grade
   end
 
+  # Sections can be assigned to both courses and scripts. We want to make sure
+  # we give teacher dashboard the same information for both sets of assignables,
+  # which we accomplish via this shared method
+  def self.assignable_info(course_or_script, hidden=false)
```

Comment:
FWIW, most (if not all) of the logic in this file probably better belongs in a dashboard controller, but moving it is a non-trivial effort and out of scope for the time being.